### ISSUE TITLE
make possible to annotate paramet for descriptions

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-plugin-nb2workflow#53]

**Comment by dsavchenko** [comment]
Do you remember what this issue is about?

**Comment by volodymyrss** [comment]
it's to add some additional property to parameter annotations in the notebook which will be used in the frontend as short description, a sentence

**Comment by dsavchenko** [comment]
`oda:label` ontology term is proposed in https://github.com/oda-hub/nb2workflow/pull/102

**Comment by burnout87** [comment]
talking with @dsavchenko , it'd be useful to introduce the concept of "group", so that those parameters annotated with the same group will be grouped together in the frontend

**Comment by dsavchenko** [comment]
Yes, this all is related in a sense that all these annotations affect the representation and all require adaptations to dispatcher to be reflected in the parameter metadata. 

So my proposal for this is to keep there the full parameter description as returned by backend so we account for any other possible extension with minimal dispatcher modifications.

**Comment by dsavchenko** [comment]
Even more broad, can we invent an annotations to express e.g. "conditional parameters" and other possible types of interrelations?
